neurokit2/complexity/utils.py

```diff
@@ -111,7 +111,7 @@ def _get_count(
     # -------------------
     # Sanity checks
     sklearn_version = version.parse(sklearn.__version__)
-    if sklearn_version >= version.parse("1.3.0"):
+    if sklearn_version in [version.parse("1.3.0"), version.parse("1.3.0rc1")]:
```

Comment:
shouldn't this be:

```python
if sklearn_version >= version.parse("1.3.0") or sklearn_version == version.parse("1.3.0rc1"):
```

Reply:
I think it does the same, I was just lazy and didn't want to write sklearn_version twice. I can change it if that's the practice.

Reply:
I see, makes sense then thanks!